presto-tests/src/test/java/io/prestosql/tests/TestLocalQueries.java

```diff
@@ -142,4 +142,14 @@ public void testIOExplain()
                 jsonCodec(IoPlan.class).fromJson((String) getOnlyElement(result.getOnlyColumnAsSet())),
                 new IoPlan(ImmutableSet.of(input), Optional.empty()));
     }
+
+    @Test
+    public void testHueQueries()
```

Comment:
Similarly, how about changing to `testMySqlInformationSchemaCompatibility` and removing the Hue links

Reply:
Makes sense.

Reply:
So I am going to keep it as it is.